### DISCUSSION TITLE
Keep alive for popover?

### DISCUSSION THREAD

**Discussion by jacobg** [bootstrap-vue/bootstrap-vue#6502] [Q&A]
Is there a way to cache a popover component once it's been opened the first time? That is, when the popover is subsequently hidden and re-opened, I'd like it to re-use the cache popover component rather than re-create it. When the popover's parent component is destroyed, then it can also destroy the popover.

Thanks.

**Comment by EverchronGreg** [comment]
https://github.com/bootstrap-vue/bootstrap-vue/discussions/6549#discussioncomment-610509

Sorry if this isn't entirely relevant to your question, but thought it may be helpful to you so figured I'd share in case
